Partners/support-escalation-process-for-partners.md

```diff
@@ -199,22 +196,12 @@ These steps sit inside the “Restoration Process” within the overall Incident
     <tr>
       <td>Management Escalation</td>
       <td>On-call Management Team</td>
-      <td>escalation@t3n.zendesk.com</td>
-    </tr>
-    <tr>
-      <td>Security SME Engagement</td>
-      <td>On-call Security SME</td>
-      <td>security@t3n.zendesk.com</td>
-    </tr>
-    <tr>
-      <td>Security Management Escalation (Warning – Do Not Abuse)</td>
-      <td>Security Management Engagement</td>
-      <td>sec-esc@t3n.zendesk.com + Escalation ticket #</td>
+      <td>escalation@ctl.io</td>
     </tr>
     <tr>
       <td>Executive Escalation (Warning – Do Not Abuse)</td>
       <td>Executive Engagement – Enterprise Support Level Only</td>
-      <td>exec-esc@t3n.zendesk.com + Management Escalation ticket #</td>
+      <td>execescalation@ctl.io + Management Escalation ticket #</td>
```

Comment:
not sure this should be public,  escalation is done via a formal process not customer emailing executives (which pages them) directly.  please align with support process.